ScriptureRenderingPipeline/Renderers/BibleRenderer.cs

```diff
@@ -33,7 +34,7 @@ public static class BibleRenderer
 		/// <param name="textDirection">The direction of the script being used (either rtl or ltr)</param>
 		/// <param name="isBTTWriterProject">Whether or not this is a BTTWriter project</param>
 		/// <param name="languageCode">The language code for the project</param>
-		public static async Task RenderAsync(ZipFileSystem source, string basePath, string destinationDir, Template printTemplate, string repoUrl, string heading, string languageCode, string languageName, string textDirection, bool isBTTWriterProject = false)
+		public static async Task RenderAsync(ZipFileSystem source, string basePath, string destinationDir, Template printTemplate, string repoUrl, string heading, string languageCode, string languageName, string textDirection, bool isBTTWriterProject = false, JsonElement appsMeta = new JsonElement())

```

Comment:
Each parameter should go on its own line, especially since there are default parameters here.

There's also a lot of parameters in general, it would be worth considering if parameters here can/should be grouped into objects.